Jellyfin.Api/ModelBinders/CommaDelimitedArrayModelBinder.cs

```diff
@@ -20,30 +33,17 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
 
             if (valueProviderResult.Length > 1)
             {
-                var result = Array.CreateInstance(elementType, valueProviderResult.Length);
-
-                for (int i = 0; i < valueProviderResult.Length; i++)
-                {
-                    var value = converter.ConvertFromString(valueProviderResult.Values[i].Trim());
-
-                    result.SetValue(value, i);
-                }
-
-                bindingContext.Result = ModelBindingResult.Success(result);
+                var typedValues = GetParsedResult(valueProviderResult.Values, elementType, converter);
+                bindingContext.Result = ModelBindingResult.Success(typedValues);
             }
             else
             {
                 var value = valueProviderResult.FirstValue;
 
                 if (value != null)
                 {
-                    var values = Array.ConvertAll(
-                        value.Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries),
-                        x => converter.ConvertFromString(x?.Trim()));
-
-                    var typedValues = Array.CreateInstance(elementType, values.Length);
-                    values.CopyTo(typedValues, 0);
-
+                    var splitValues = value.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
```suggestion
                    var splitValues = value.Split(',', StringSplitOptions.RemoveEmptyEntries);
```
Don't allocate an array for this ;)